pkg/service/model_build_target_group.go

```diff
@@ -0,0 +1,312 @@
+package service
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"regexp"
+	elbv2api "sigs.k8s.io/aws-load-balancer-controller/apis/elbv2/v1beta1"
+	"sigs.k8s.io/aws-load-balancer-controller/pkg/annotations"
+	elbv2model "sigs.k8s.io/aws-load-balancer-controller/pkg/model/elbv2"
+	"strconv"
+	"strings"
+)
+
+const (
+	TGAttrsProxyProtocolV2Enabled  = "proxy_protocol_v2.enabled"
+	TGAttrsPreserveClientIPEnabled = "preserve_client_ip.enabled"
```

Comment:
seems this `preserve_client_ip ` feature isn't released before reinvent. better to remove it from the code.

Reply:
I'd prefer to put in place since beta customers can still utilize the feature. Unlike other attributes, this one has a dependency on how the security group rules are constructed.

Reply:
I would +1 this... we are trying to get in on the beta to test this feature